builtin/credential/okta/path_login.go

```diff
@@ -117,7 +122,8 @@ func (b *backend) pathLoginRenew(ctx context.Context, req *logical.Request, d *f
 		return nil, err
 	}
 
-	loginPolicies, resp, groupNames, err := b.Login(ctx, req, username, password)
+	// No TOTP entry is possible on renew. If push MFA is enabled it will still be triggered, however.
```

Comment:
So what happens if push MFA is enabled and a renewal happens since we are passing in an empty passcode?

Reply:
You get a push notification.

Reply:
Can you make note of this behavior in the comments here?

Reply:
Updated